src/test/java/org/example/stepsDefinitions/LoginFunctionalitiesSteps.java

```diff
@@ -0,0 +1,30 @@
+package org.example.stepsDefinitions;
+
+import io.cucumber.java.en.*;
+
+
+public class LoginFunctionalitiesSteps {
+	@Given("the user has navigated to the login page")
+	public void theUserHasNavigatedToTheLoginPage() {
+
+	}
+
+	@When("the user enters the correct username and password")
```

Comment:
Use args/params in methods.
The user loging with username: {string} and password: {string}.